### ISSUE TITLE
Shared credentials

### ISSUE THREAD

**Issue by dominykas** [nodejs/package-maintenance#441]
As part of work on `wiby`, I'll need to use a bot account - I do not want to add my personal token to the repo secrets, because there is no way to restrict tokens to a subset of repos.

`wiby` needs push (only push, not merge) access to the dependent repos (or their forks) to be able to kick off the tests (i.e. the test repos under the `wiby-test` org). We'll also be able to use that token to kick off the integration tests.

I figure that I should probably not be the only one with access to that bot account - but that leaves us with a problem of sharing credentials.

Any opinions on how to best do this? Would folks trust [sops](https://github.com/mozilla/sops) enough to keep the encrypted secrets in a repo? Possibly even a public one? I keep my PGP key on a Yubikey, not sure what everyone else does.

If we can make some decisions here, we can maybe also document some practices for others?

**Comment by ljharb** [comment]
Seems reasonable to make a bot account in the pkgjs org, and make a token for it?

**Comment by mhdawson** [comment]
The build wg uses [dotgpg](https://github.com/ConradIrwin/dotgpg) to protect secrets but a private repo is used as well.

**Comment by wesleytodd** [comment]
There is an OpenJS Foundation lastpass account as well I think.  This would be a great service for the foundation to offer IMO.  I seem to remember hearing that another password app had free premium accounts for OSS projects, but I don't remember the name.

**Comment by mhdawson** [comment]
@brianwarner is this something we can use the Foundation lastpass account for?

**Comment by travi** [comment]
> I seem to remember hearing that another password app had free premium accounts for OSS projects, but I don't remember the name.

this may have been the one that youre thinking of, since 1Password is one of the more popular managers: https://github.com/1Password/1password-teams-open-source

**Comment by brianwarner** [comment]
Yes definitely, find me on slack and we can coordinate on escrowing the credentials. We have LastPass Enterprise, and I can share creds out to anyone who uses the free version.

**Comment by dominykas** [comment]
> Seems reasonable to make a bot account in the pkgjs org, and make a token for it?

Are there org level bot accounts available in GH? I thought they only have real user accounts? Which means that it's not just the password that needs to be shared, but it also needs to be associated with an email...

**Comment by mhdawson** [comment]
For email we can PR into https://github.com/nodejs/email/blob/master/iojs.org/aliases.json a new alias called:

package-maintenance-admin-members for an email.
